src/stan/lang/grammars/block_var_decls_grammar_def.hpp

```diff
@@ -1,52 +1,52 @@
 #ifndef STAN_LANG_GRAMMARS_BLOCK_VAR_DECLS_GRAMMAR_DEF_HPP
 #define STAN_LANG_GRAMMARS_BLOCK_VAR_DECLS_GRAMMAR_DEF_HPP
 
+#include <boost/phoenix/phoenix.hpp>
+#include <boost/spirit/include/qi.hpp>
+#include <boost/version.hpp>
 #include <stan/lang/ast.hpp>
 #include <stan/lang/grammars/block_var_decls_grammar.hpp>
 #include <stan/lang/grammars/common_adaptors_def.hpp>
 #include <stan/lang/grammars/semantic_actions.hpp>
-#include <boost/spirit/include/qi.hpp>
-#include <boost/phoenix/phoenix.hpp>
-#include <boost/version.hpp>
-#include <sstream>
 #include <set>
+#include <sstream>
 #include <string>
 #include <vector>
 
```

Comment:
looks like auto-format has done funky things here - can this be corrected somehow?